crates/forge/tests/cli/cmd.rs

```diff
@@ -666,6 +666,45 @@ contract ATest is DSTest {
     }
 );
 
+// test that `forge build` does not print `(with warnings)` if file path is ignored
+forgetest!(can_compile_without_warnings_ignored_file_paths, |prj, cmd| {
+    let create_config =
+        |ignored_file_paths: Vec<PathBuf>, ignored_error_codes: Vec<SolidityErrorCode>| -> Config {
+            Config { ignored_file_paths, ignored_error_codes, ..Default::default() }
+        };
```

Comment:
pedantic nitpick: we don't really need this, should just create `Config {.., Default::default}` where used